### ISSUE TITLE
[Bug] Fluid Transfer Issue

### ISSUE THREAD

**Issue by AoSora98** [SuperMartijn642/Tesseract#26]
Im having an issue with the fluid transfer option on v1.0.20 for 1.16.5.
Fluids will just not transfer though the tesseracts. a channel (fluid tab) is created and selected on both tesseracts, the pipes get also conected and their direction is setted correctly, but the tesseracts won't transfer any liquid

**Comment by SuperMartijn642** [comment]
It might be the same issue as #15, but just with fluids. Placing a tank against the tesseract at one end should fix that.
What pipes are you using?

**Comment by AoSora98** [comment]
I used 3 different mods thinking it was a problem with the pipes: Pretty pipes fluids, refined pipes and large fluid tank pipes. Y tryed placing the tesseract against a tank and it recieve the liquid without a problem, still, a pipe dumping the liquid into the tesseract is needed. I did some additional tests and sometimes the pipe recieve the liquid from the tesseract and others it doesn't. Don't know why though. Thanks for the help!
